docs/user_guide/how_to_xxx.md

```diff
@@ -0,0 +1,30 @@
+# How to xxx
```

Comment:
```suggestion
# Completing a project use case
```

Reply:
Definitely prefer a gerund in the title of a "How To" that directly addresses user motivation. Plus when you start building out the doc you don't want to have 10 "How to xxx" titles stacked on top of each other. That makes it a lot harder for users to parse the nav etc.